requirements.txt

```diff
@@ -0,0 +1,20 @@
+pillow>=4.3.0
```

Comment:
do we need this file?

Reply:
Let's keep for now, and revisit when we finalize the container process for 20.01